runtime/queries/column_time_range.go

```diff
@@ -127,50 +127,72 @@ func handleDuckDBInterval(interval any) (*runtimev1.TimeRangeSummary_Interval, e
 }
 
 func (q *ColumnTimeRange) resolveDruid(ctx context.Context, olap drivers.OLAPStore, priority int) error {
-	rangeSQL := fmt.Sprintf(
-		"SELECT min(%[1]s) as \"min\", max(%[1]s) as \"max\", timestampdiff(SECOND, min(%[1]s), max(%[1]s)) as \"interval\" FROM %[2]s",
+	minSQL := fmt.Sprintf(
+		"SELECT min(%[1]s) as \"min\" FROM %[2]s",
+		safeName(q.ColumnName),
+		safeName(q.TableName),
+	)
+	maxSQL := fmt.Sprintf(
+		"SELECT max(%[1]s) as \"max\" FROM %[2]s",
 		safeName(q.ColumnName),
 		safeName(q.TableName),
 	)
 
-	rows, err := olap.Execute(ctx, &drivers.Statement{
-		Query:            rangeSQL,
+	minRows, err := olap.Execute(ctx, &drivers.Statement{
+		Query:            minSQL,
 		Priority:         priority,
 		ExecutionTimeout: defaultExecutionTimeout,
 	})
 	if err != nil {
 		return err
 	}
-	defer rows.Close()
+	defer minRows.Close()
 
-	if rows.Next() {
-		summary := &runtimev1.TimeRangeSummary{}
-		rowMap := make(map[string]any)
-		err = rows.MapScan(rowMap)
+	maxRows, err := olap.Execute(ctx, &drivers.Statement{
+		Query:            maxSQL,
+		Priority:         priority,
+		ExecutionTimeout: defaultExecutionTimeout,
+	})
+	if err != nil {
+		return err
+	}
+	defer maxRows.Close()
+
+	summary := &runtimev1.TimeRangeSummary{}
+
+	if minRows.Next() && maxRows.Next() {
+		minRowMap := make(map[string]any)
+		err = minRows.MapScan(minRowMap)
 		if err != nil {
 			return err
 		}
-		if v := rowMap["min"]; v != nil {
+		maxRowMap := make(map[string]any)
+		err = maxRows.MapScan(maxRowMap)
+		if err != nil {
+			return err
+		}
```

Comment:
Use regular `Scan` instead of `MapScan` to scan directly into a `time.Time`. Gets rid of map and time casting.